### PR TITLE
bugfix for loss joint proj

### PR DESCRIPTION
**Pull request by bobopit**
It seems that joint_proj loss uses the wrong weight(`meta["joint_valid"]`). When the detection box is inaccurate, some key points are outside the detection box. These points should not be supervised. The correct weight may be `meta["joint_trunc"]`
https://github.com/facebookresearch/InterWild/blob/1a0da99386f60fad7cba802d0188fcf6b9fff57f/main/model.py#L160

### CODE REVIEW

**Comment by mks0601** [comment]
Using joint_valid is correct. That loss function is computed by projecting 3D data. Although some projected 2D coordinates could be outside of the bbox, it doesn't matter.

**Comment by bobopit** [comment]
When the detection frame is inaccurate, for example, the object in the detection frame is a yellow kraft paper bag, the network will predict a meaningless mesh and project it back to the 2D space through this mesh. At this time, it is meaningless to use weak supervision.

**Comment by bobopit** [comment]
When the detection box is inaccurate, for example, the object in the detection box is a yellow kraft paper bag, the network will predict a meaningless mesh and project it back to the 2D space through this mesh. At this time, it is meaningless to use weak supervision.

**Comment by mks0601** [comment]
yeap in that case, you're correct, in empirically, I found that using joint_valid gives better results as in the training stage, most of the hand boxes are correct. Or have you observed that using joint_trunc gives better results?

**Comment by bobopit** [comment]
Yes, using joint_trunc can get better qualitative results. Since I added my own data during the training phase, I cannot give quantitative results on public benchmarks such as HIC. However, using joint_trunc can improve the loss of joint_proj compared to joint_valid, obtain smoother results, and the values are in a more reasonable range

**Comment by mks0601** [comment]
I see. Thanks! I've included it
